pages/_app.js

```diff
@@ -1,14 +1,25 @@
 import GlobalStyle from "@/styles";
 import Head from "next/head";
+import { SWRConfig } from "swr";
 
 export default function App({ Component, pageProps }) {
   return (
-    <>
+    <SWRConfig
+      value={{
+        fetcher: async (...args) => {
+          const response = await fetch(...args);
+          if (!response.ok) {
+            throw new Error("Ein Fehler ist aufgetreten.");
```

Comment:
I tried the preview on vercel and it says "ein fehler ist aufgetreten" i dont know if you have even data on your database or if there is something wrong. Maybe you should make that clear . also it will not load the package-lock.json so that I can't look inside .

Reply:
About the package-lock.json don't worry about it. It's a whole bunch of crypto things, since a add the import to my backend. 
But if you are interested you can either press `.` in review view or look inside the the code itself in the branch view.

Reply:
- [x] FIX env variables in vercel